src/Controller/ApiController.php

```diff
@@ -189,7 +189,7 @@ private function fetchDownloaded(array $media)
 
         $downloaded = [];
 
-        foreach ($stmt->fetchAll() as $row) {
+        foreach ($stmt->fetchAllAssociative() as $row) {
```

Comment:
We should also add the doctrine dbal dependency to our `composer.json` in order to prevent possible issues here.